openpype/hosts/maya/api/lib.py

```diff
@@ -3928,7 +3928,7 @@ def get_capture_preset(task_name, task_type, subset, project_settings, log):
     }
 
     plugin_settings = project_settings["maya"]["publish"]["ExtractPlayblast"]
-    if plugin_settings["profiles"]:
+    if plugin_settings.get("profiles", []):
```

Comment:
```suggestion
    if plugin_settings["profiles"]:
```

There is a fix coming for the addon settings [here](https://github.com/ynput/ayon-addons/pull/10)